File: 2st/Yeonwoo/최빈값 구하기.py
def solution(array):
    answer = 0
    max_ = 0
    temp2 = []
    temp = set(array)
    for i in temp:
        if array.count(i) > max_:
            max_ = array.count(i)
            answer = i
        elif array.count(i) == max_:
            answer = -1
    return answer

File: 5st/gyuri/최빈값 구하기.java
class Solution {
    public int solution(int[] array) {
        int max = 0;
        int num = 0;
        int answer = 0;
        
        int count[] = new int[1000];
        for (int n : array){
            count[n]++;
        }
        
        for(int c : count ){
            if( c > max)
                max = c;
        }
        
        for (int i = 0; i < count.length; i++){
            if (count[i] == max){
                answer = i;
                num++;
            }
        }
        if(num > 1)
            answer = -1;
        
        return answer;
    }
}